python/pyspark/sql/functions.py

```diff
@@ -2804,6 +2804,22 @@ def pandas_udf(f=None, returnType=None, functionType=None):
        |  1|1.5|
        |  2|6.0|
        +---+---+
+       >>> @pandas_udf(
+       ...    "id long, additional_key double, v double",
```

Comment:
do you mind changing the type of additional_key to long? It seems like the type coercion here is not necessary.

Reply:
Sorry, I know you just changed it, but I think just naming the column "ceil(v1 / 2)" with a type `long` would be a little more clear. Although "additional_key" is ok too, if you guys want to keep that.